docs/how_to_ignore.md

```diff
@@ -26,3 +33,17 @@ def my_function(  # noqa: DOC2
 All the usage is consistent with how you would use _flake8_. Please read the
 official _flake8_ documentation for full details:
 https://flake8.pycqa.org/en/latest/user/violations.html.
+
+### Usage with [`ruff`](https://github.com/astral-sh/ruff)
```

Comment:
```suggestion
### Usage with [`Ruff`](https://github.com/astral-sh/ruff)
```

Reply:
I would rather have one of these two:

- Ruff (no code highlighting)
- `ruff` (matching command line)

I will move to "Ruff" not "`Ruff`"